### PR TITLE
Fixes #82 Moved docs source from Fluo repo to website

### LINKED ISSUE #82: Move Fluo documentation source to fluo-website repo

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: _fluo-docs-1-2/development/applications.md [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+The log above traces the following sequence of events.
+
+* Transaction T1 has a start timestamp of `3`
+* Thread with id `198` is executing T1, its running code from the class `com.SimpleLoader`
+* T1 reads row `4333` and column `stat count` which does not exist
+* T1 sets row `4333` and column `stat count` to `1`
+* T1 commits successfully and its commit timestamp from the Oracle is `4`.
+* Transaction T2 has a start timestamp of `5` (because its `5` > `4` it can see what T1 wrote).
+* T2 reads a value of `1` for row `4333` and column `stat count`
+* T2 sets row `4333` and `column `stat count` to `2`
```
Comment (keith-turner) [inline _fluo-docs-1-2/development/applications.md]: This is bug that was copied over. I noticed it while looking at this rendered through jekyll.  There is an extraneous back tick in front of column.

File: _layouts/fluo-docs-1.2.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+skiph1fortitle: true
+---
+
+<div class="row">
```
Comment (keith-turner) [inline _layouts/fluo-docs-1.2.html]: This sidebar is really slick.

File: _fluo-docs-1-2/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Apache Fluo documentation
+nodoctitle: true
+---
+
+# Apache Fluo {{ page.version }} documentation
```
Comment (keith-turner) [inline _fluo-docs-1-2/index.md]: The following concept is not fully formed, just sharing for consideration and discussion.

Maybe we could remove this page and do the following.

 * Add Release notes and Javadocs to sidebar.
 * When linking to 1.2/ docs, maybe it just redirects to 1.2/design.  So when you go to the 1.2 docs it goes to the design docs and that shows as open in the sidebar.
 * The Fluo Tour link is that the top of every Fluo page.. so maybe a Tour link is not needed.

File: _config.yml [hunk 4]
```diff
@@ -34,6 +40,19 @@ defaults:
     values:
       layout: "tour"
       permalink: "/tour/:basename/"
+  -
+    scope:
+      path: ""
```
Comment (keith-turner) [inline _config.yml]: Can you explain these changes?  I am not sure how they are used. Why is `path` empty?  What is `skiph1fortitle`?  I suspect this section is somehow for 1.2 doc specific vars, but not sure.

File: _config.yml [hunk 5]
```diff
@@ -34,6 +40,19 @@ defaults:
...
+      type: "fluo-docs-1-2"
+    values:
+      layout: "fluo-docs-1.2"
+      title_prefix: "Fluo Documentation - "
+      version: "1.2.0"
+      minor_release: "1.2"
+      docs_base: "/docs/fluo/1.2"
+      javadoc_base: "https://static.javadoc.io/org.apache.fluo/fluo-api/1.1.0-incubating"
+      github_base: "https://github.com/apache/fluo/blob/master"
```
Comment (keith-turner) [inline _config.yml:54]: Should this be a tag?  The 1.2 tag eventually?  If so, maybe we can open an issue or set it now?

File: _fluo-docs-1-2/administration/metrics.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+5.  If you restart Grafana, you will see the Fluo dashboards configured but all of their charts will
+    be empty unless you have a Fluo application running and configured to send data to InfluxDB.
+    When you start sending data, you may need to refresh the dashboard page in the browser to start
+    viewing metrics.
+
+[Grafana]: http://grafana.org/
+[InfluxDB]: https://influxdb.com/
+[fluo-app.properties]: {{ site.github_base }}/modules/distribution/src/main/config/fluo-app.properties
+[Dropwizard]: https://dropwizard.github.io/metrics/3.1.0/
+[grafana]: {{ site.docs_base}}/administration/grafana/
```
Comment (keith-turner) [inline _fluo-docs-1-2/administration/metrics.md]: Can this be removed?

File: _fluo-docs-1-2/administration/metrics.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+[MetricsReporter]: {{ site.github_base }}/modules/api/src/main/java/org/apache/fluo/api/metrics/MetricsReporter.java
```
Comment (keith-turner) [inline _fluo-docs-1-2/administration/metrics.md]: Seems like these should point to javadocs.

File: _fluo-docs-1-2/development/applications.md [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
+2015-02-11 18:17:02,639 [tx.collisions] TRACE: txid: 106 class: com.SimpleLoader
+2015-02-11 18:17:02,639 [tx.collisions] TRACE: txid: 106 collisions: {r1=[fam1 qual1 ]}
+```
+
+When applications read and write arbitrary binary data, this does not log so well. In order to make
+the trace logs human readable, non ASCII chars are escaped using hex. The convention used it `\xDD`
+where D is a hex digit. Also the `\` character is escaped to make the output unambiguous.
+
+[design]: {{ page.docs_base }}/getting-started/design/
+[FluoFactory]: {{ page.github_base}}/modules/api/src/main/java/org/apache/fluo/api/client/FluoFactory.java
```
Comment (keith-turner) [inline _fluo-docs-1-2/development/applications.md]: Seems like this should point to javadoc.

File: _fluo-docs-1-2/getting-started/design.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
+---
+title: Design
+category: getting-started
+order: 1
+---
+
+For a general overview of Fluo, take the [Fluo tour](/tour/). The diagram below provides an
```
Comment (keith-turner) [inline _fluo-docs-1-2/getting-started/design.md]: Not sure this link to Fluo tour is needed, but it doesn't hurt.

**Review by keith-turner — COMMENTED** [review]
Why are docs in a `_fluo-docs-1-2` dir and not in `docs/fluo/1.2.0`?  Is this to keep them from showing up in menus for now and we will move them are release?

File: _fluo-docs-1-2/development/applications.md (hunk 1 above)
Comment (mikewalch) [reply _fluo-docs-1-2/development/applications.md]: Fixed

File: _fluo-docs-1-2/development/applications.md (hunk 8 above)
Comment (mikewalch) [reply _fluo-docs-1-2/development/applications.md]: I fixed all javadoc references in d857181cc4

File: _config.yml (hunk 4 above)
Comment (mikewalch) [reply _config.yml]: Path shouldn't be empty.  Also skiph1fortitle can be removed.  This should be fixed now.

File: _fluo-docs-1-2/getting-started/design.md (hunk 9 above)
Comment (mikewalch) [reply _fluo-docs-1-2/getting-started/design.md]: Removed the link.

File: _config.yml (hunk 5 above)
Comment (mikewalch) [reply _config.yml:54]: Created #88

File: _fluo-docs-1-2/administration/metrics.md (hunk 6 above)
Comment (mikewalch) [reply _fluo-docs-1-2/administration/metrics.md]: Yes. It's removed now.

File: _fluo-docs-1-2/administration/metrics.md (hunk 7 above)
Comment (mikewalch) [reply _fluo-docs-1-2/administration/metrics.md]: Fixed

**Comment by mikewalch** [comment]
> Why are docs in a _fluo-docs-1-2 dir and not in docs/fluo/1.2.0? Is this to keep them from showing up in menus for now and we will move them are release?

Collections need to exists in directory at root that starts with underscore.

**Review by keith-turner — APPROVED** [review]

**Pull request merged**